### PR TITLE
topic auth docs added

### PR DESCRIPTION
**Pull request by wojtkiewicz**
* added docs and changed
* topic.auth.unauthorisedAccessEnabled ->  topic.auth.unauthenticatedAccessEnabled

### CODE REVIEW

File: docs/docs/user/permissions.md [hunk 1]
```diff
@@ -26,11 +26,44 @@ Add new topic         | any logged in user
 Remove existing topic | **topic owner**
 Modify topic          | **topic owner**
 
+### Publishing permission
+
+You can configure which services can publish on which topic configuring **topic.auth** section.
+How publisher name is evaluated is deployment specific. It can be extracted from ssl certificate or read from supplied header for instance.
```
Comment (adamdubiel) [inline docs/docs/user/permissions.md]: add note that for it to work atuh has to be enabled by Hermes administrator

Comment (adamdubiel) [inline docs/docs/user/permissions.md]: and add info in `Configuration` section how to do it

Comment (druminski) [inline docs/docs/user/permissions.md]: Can I find in docs information about how to configure publisher name evaluation? Does Hermes provide some kind of default behaviour for this? Imo it is worth to mention about it here

Comment (wojtkiewicz) [inline docs/docs/user/permissions.md]: done

Comment (wojtkiewicz) [inline docs/docs/user/permissions.md]: added publishing-security section

File: docs/docs/configuration/publishing-security.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+# Publishing security
+
+Hermes has a feature to restrict publishing on specific topics to particular services: [publishing permissions](/user/permissions)
```
Comment (dankraw) [inline docs/docs/configuration/publishing-security.md]: Missing period at the end of sentence.

**Pull request merged**